.travis.yml

```diff
@@ -35,11 +35,14 @@ before_install:
 - nvm install $NODE_VERSION
 - nvm use $NODE_VERSION
 before_script:
+- npm install -g jsinspect
 - node -e 'require("./lib/index.js")'
 - psql -c 'create database parse_server_postgres_adapter_test_database;' -U postgres
 - psql -c 'CREATE EXTENSION postgis;' -U postgres -d parse_server_postgres_adapter_test_database
 - psql -c 'CREATE EXTENSION postgis_topology;' -U postgres -d parse_server_postgres_adapter_test_database
 - silent=1 mongodb-runner --start
+script:
+- jsinspect ./src && npm test
```

Comment:
we could run in before_script right? any failure would be caught.

Reply:
Yeah we can do that if we don't want to fudge with just `script`.